### PR TITLE
fix permission of yq binary

### PR DESCRIPTION
**Pull request by umezawatakeshi**
part of https://github.com/cybozu-go/neco/issues/1956

Signed-off-by: UMEZAWA Takeshi <takeshi-umezawa@cybozu.co.jp>

### CODE REVIEW

File: Makefile.tools [hunk 1]
```diff
@@ -47,6 +47,9 @@ HUBBLE_DOWNLOAD := $(HUBBLE_DLDIR)/downloaded
 
 .PHONY: all
 all: node_exporter containerd crictl runc argocd kubectl kubeseal stern kustomize teleport cke moco logcli yq kubectl-neat accurate alertmanager cilium-cli hubble
+	# all binaries in BINDIR and SBINDIR must have executable bits
+	test `find $(BINDIR) -type f -a \( ! -perm -111 \) | wc -l` = 0
+	test `find $(SBINDIR) -type f -a \( ! -perm -111 \) | wc -l` = 0
```
Comment (morimoto-cybozu) [inline Makefile.tools:52]: `$$(...)` would be better for appearance.

```suggestion
	test -z "$$(find $(BINDIR) $(SBINDIR) -type f -a \( \! -perm -111 \) | tee /dev/stderr)"
```

**Review by morimoto-cybozu — CHANGES_REQUESTED** [review]

**Review by morimoto-cybozu — APPROVED** [review]
LGTM
`artifacts.go` requires update.  Please rebase after the branch promtail-v2.5.0 is merged.

File: Makefile.tools [hunk 2]
```diff
@@ -47,6 +47,8 @@ HUBBLE_DOWNLOAD := $(HUBBLE_DLDIR)/downloaded
 
 .PHONY: all
 all: node_exporter containerd crictl runc argocd kubectl kubeseal stern kustomize teleport cke moco logcli yq kubectl-neat accurate alertmanager cilium-cli hubble
+	# all binaries in BINDIR and SBINDIR must have executable bits
+	test -z $$(find $(BINDIR) $(SBINDIR) -type f -a \( ! -perm -111 \) | tee /dev/stderr)
```
Comment (morimoto-cybozu) [inline Makefile.tools:51]: The operator for `test -z` should be one string.
Try `test -z $(ls -a); echo $?` and `test -z "$(ls -a)"; echo $?` for example.
Though the results are both evaluated to False, the former one produces an error.

It is recommended to escape `!`.

```suggestion
	test -z "$$(find $(BINDIR) $(SBINDIR) -type f -a \( \! -perm -111 \) | tee /dev/stderr)"
```

**Review by morimoto-cybozu — CHANGES_REQUESTED** [review]

File: Makefile.tools (hunk 2 above)
Comment (umezawatakeshi) [reply Makefile.tools:51]: There is no excuse for what I have (not) done...

**Pull request merged**